modules/ipc/methods/eth_sendTransaction.js

```diff
@@ -36,14 +36,16 @@ module.exports = class extends BaseProcessor {
             try {
                 _.each(payload.params[0], (val, key) => {
                     // if doesn't have hex then leave
-                    if (_.isString(val)) {
+                    if (_.isString(val) || _.isNumber(val)) {
 
                         // make sure all data is lowercase and has 0x
                         if (val) val = `0x${val.toLowerCase().replace('0x', '')}`;
```

Comment:
This isn't part of the PR, I know... but doing `toLowerCase()` here removes the checksum (if any), and could be problematic later on if you want to add checksum verifications..